main_test.go

```diff
@@ -550,7 +550,7 @@ func TestRotate(t *testing.T) {
 				panic(err)
 			}
 			for idx, row := range tc.PasswordManagerIn {
-				err := f.SetSheetRow(sheetNamePasswordManager, "A"+string(rune('2'+idx)), &[]string{
+				err := f.SetSheetRow(sheetNamePasswordManager, "A"+fmt.Sprintf("%d", 2+idx), &[]string{
```

Comment:
nit: `fmt.Sprintf("A%d", 2+idx)` looks nicer to me, but leave it as-is if you prefer

Reply:
Agree. Done.